app/controllers/reports_controller.rb

```diff
@@ -2,13 +2,13 @@ class ReportsController < ApplicationController
   def create
     report = Report.new (report_params)
     if report.save 
-      flash[:notice] = "Post reported"
-      respond_to do |format|
-        format.js
-        format.html {  redirect_to post_path(report.post_id, report.post) }
-      end
+      flash[:notice] = "Post is successfully reported"
```

Comment:
Please improve the message. Use something like, "Thanks for taking the time to report this post. Admins will be notified about this. "

Also send emails to Admins.

Reply:
The message is improved